### PR TITLE
storeservice: save on server's command.

### PR DESCRIPTION
**Pull request by deekoder**
- minor : separate commits for space removals in files.

### CODE REVIEW

File: app/src/main/java/com/minio/io/alice/ClientWebSocket.java [hunk 1]
```diff
@@ -47,9 +47,7 @@ public void connect(Context context) {
             Replace with the IP address of local xray server
             if needed.
         */
-
         final String wsuri = "ws://147.75.201.195:80";
-
         final WebSocketOptions webSocketOptions = new WebSocketOptions();
         webSocketOptions.setMaxMessagePayloadSize(100 * 1024 * 1024);
```
Comment (deekoder) [inline app/src/main/java/com/minio/io/alice/ClientWebSocket.java]: Removed some unnecessary whitespace in some of the files.

File: app/src/main/java/com/minio/io/alice/IncomingReceiver.java [hunk 2]
```diff
@@ -39,9 +39,10 @@ public IncomingReceiver() {
...
-        if(XDebug.LOG)
-            Log.i(MainActivity.TAG, msg);
-        MainActivity.serverReply = new XPly(msg);
-
+        if (msg != null) {
+            if(XDebug.LOG)
+                Log.i(MainActivity.TAG, msg);
+            MainActivity.serverReply = new XPly(msg);
+        }
     }
```
Comment (deekoder) [inline app/src/main/java/com/minio/io/alice/IncomingReceiver.java]: Prevent client from processing null messages from server.

File: app/src/main/java/com/minio/io/alice/StoreService.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    MinioClient minioClient;
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+
+        // TODO: These params could come from the xray server. Make this configurable.
+        try {
+            minioClient = new MinioClient("https://play.minio.io:9000", "Q3AM3UQ867SPQQA43P2F", "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG");
```
Comment (deekoder) [inline app/src/main/java/com/minio/io/alice/StoreService.java]: Will make this configurable.

File: app/src/main/java/com/minio/io/alice/StoreService.java [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+                minioClient.makeBucket("alice");
+            }
+        } catch(Exception e) {
+            e.printStackTrace();
+        }
+
+        // Create an object.
+        try {
+
+            minioClient.putObject("alice", String.valueOf(new java.util.Date()), inputStream, inputStream.available(), "image/png");
```
Comment (deekoder) [inline app/src/main/java/com/minio/io/alice/StoreService.java]: Will make bucket name configurable next.

File: app/src/main/java/com/minio/io/alice/XDebug.java [hunk 5]
```diff
@@ -24,8 +24,6 @@
 // On off switch for enabling log messages. Filter out in android studio with "__ALICE__"
 
 public class XDebug {
-
+    // set this to true during development only.
     public  static  final boolean LOG = false;
-
-
```
Comment (deekoder) [inline app/src/main/java/com/minio/io/alice/XDebug.java]: remove white space. Add a comment when to turn this on to true/false.

File: app/src/main/java/com/minio/io/alice/MainActivity.java [hunk 6]
```diff
@@ -285,11 +292,12 @@ public Mat onCameraFrame(CvCameraViewFrame inputFrame) {
             return blackMat;
         }
     }
-
+    // Use Hocket Framework to collect crash reports on clients.
```
Comment (poornas) [inline app/src/main/java/com/minio/io/alice/MainActivity.java]: typo
